ismip6_ocean_forcing/woa/main.py

```diff
@@ -25,7 +25,8 @@ def process_woa(config, decades):
             baseURL = 'https://data.nodc.noaa.gov/thredds/fileServer/ncei/' \
                       'woa/{}/{}/0.25/'.format(fieldName, decade)
             fileNames = ['woa18_{}_{}00_04.nc'.format(decade, shortName)]
-            download_files(fileNames, baseURL, 'woa')
+            if not os.path.isfile(os.path.join('woa',fileNames[0])):
```

Comment:
Let's check for the last file, not the first one:
```suggestion
            if not os.path.exists(os.path.join('woa',fileNames[-1])):
```